schemas/tlc/1.3.0/sxl.yaml

```diff
@@ -114,46 +114,68 @@ objects:
+
+          The base cycle counter is based on just the internal clock and the
+          cycle length. Controllers with synchronized clocks and the same cycle
+          length will therefore have identical base cycle counters.
+
+          The cycle clock is based on the base cycle counter, but shifts the
+          phase using the offset set in the controller:
+
+          c = (b + o) modulo t
+
+          where:
+          c = cycle counter
+          b = base cycle counter
+          o = offset
+          t = cycle length
+
+          See the coordination section for more information about cycle
+          counters.
+
+          The millisecond attributes updates only when the
+          signal group state changes, meaning it's safe to subscribe to it
+          using **sendOnChange** - you will not be flooded with state updates
+          every millisecond.
         from_version: 1.0.0
         arguments:
           signalgroupstatus:
             type: string
             description: |-
               Signal group status as text field.
-              Each character represent the state of the signal group in consecutive order,
+              Each character represents the state of the signal group in consecutive order,
               where the leftmost character starts with signal group 1.
               Signal group status is described in detail in the corresponding section.
-              - : Signal group is undefined/does not exist
+              A dash "-" is used for undefined/non-exiting signal groups.
             pattern: "^[a-hA-G0-9N-P]*$"
-          cyclecounter:
+          basecyclecounter:
             type: integer_as_string
             description: |-
-              Cycle counter.
-              Used for handling of coordination between TLC’s.
-              Is counted from 0 until it reaches the cycle time (See S0028).
-              
-              c = (b + o) mod t
-              
-              where c = cycle counter,
-              b = base cycle counter,
-              o = offset,
-              t = cycle time,
-              mod = modulo
-
-              See the coordination section for more information.
+              Base cycle counter.
+              Updates once per second, counting from 0 and wrapping around at the cycle.
             min: 0
             max: 999
-          basecyclecounter:
+          cyclecounter:
             type: integer_as_string
             description: |-
-              Base cycle counter.
-              Used for handling of coordination between TLC’s.
-              Synchronized between all TLC’s in an active coordination.
-              See the coordination section for more information.
+              Cycle counter.
+              Updates once per second, counting from 0 and wrapping around at the cycle.
             min: 0
             max: 999
+          millisecond:
+            type: integer_as_string
+            min: 0
+            max: 999999
```

Comment:
"999999" seems excessive? "999"?

Reply:
the max for the cycle time is 999s, and this is 99900 milliseconds - and assuming the cycle time is actually 1000s, you can reach 999999ms